test/Bruno/Altinn.Authorization/environments/AT22.bru

```diff
@@ -3,7 +3,5 @@ vars {
   tokenEnv: at22
   tokenBasicAuthUser: {{process.env.TOKEN_BASIC_AUTH_USER}}
   tokenBasicAuthPw: {{process.env.TOKEN_BASIC_AUTH_PW}}
+  apimSubscriptionKey: {{process.env.APIM_SUBSCRIPTION_KEY}}
```

Comment:
Denne burde vel inn i alle environments om den ikke finnes. Samtidig vil det vel fungere dårlig å hente denne fra .env fil med samme navn da subscription key vil være forskjellig for hvert miljø

Reply:
Alternativt burde den vel bare ligge som en secret verdi i hver environment bru fil

Reply:
Alle disse verdiene blir hentet ut fra pipelines og overskriver det som ligger i env filen før testene kjøres. Jeg har hvertfall kunne kjørt access-management-bruno testene i alle miljøene med dette oppsettet.

Reply:
apimSubscriptionKey er nå lagt inn på lik måte i alle .bru env filer, utenom prod.

Reply:
apimSubscriptionKey er nå reflektert per miljø i .env filen, og routet riktig i hver test og {envname}.bru fil for å reflektere dette.